legend-engine-testable/src/test/java/org/finos/legend/engine/testable/assertion/TestTestAssertionEvaluator.java

```diff
@@ -152,4 +154,22 @@ public void testEqualToJsonAssertionWithDecimalPrecision()
         Assert.assertTrue(assertionStatus instanceof AssertPass);
         Assert.assertEquals("assert1", assertionStatus.id);
     }
+
+    @Test
+    public void testEqualToAssertionWithFloatPrecision() throws IOException
```

Comment:
Can we please modify the test to not get object mapper and have replicated code already present in TestAssertionHelper.
Please take a look at this test `testEqualToJsonAssertionWithDecimalPrecision` above, we should use the visitor directly and have logic there itself and just assert on output.